web/app/components/base/tag-management/selector.tsx

```diff
@@ -114,7 +114,7 @@ const Panel = (props: PanelProps) => {
     onCacheUpdate(selectedTags)
     Promise.all([
       ...(addTagIDs.length ? [bind(addTagIDs)] : []),
-      ...[removeTagIDs.length ? removeTagIDs.map(tagID => unbind(tagID)) : []],
+      removeTagIDs.length ? removeTagIDs.map(tagID => unbind(tagID)) : [],
```

Comment:
I guess the origin code want 

```js
...(removeTagIDs.length ? removeTagIDs.map(tagID => unbind(tagID)) : []),
```

instead of

```js
...[xxx]
```